src/main/java/com/jakewharton/retrofit/Ok3Client.java

```diff
@@ -41,9 +43,16 @@ public Ok3Client(Call.Factory client) {
   }
 
   static okhttp3.Request createRequest(Request request) {
+    RequestBody requestBody;
+    if (HttpMethod.requiresRequestBody(request.getMethod()) && request.getBody() == null) {
```

Comment:
Yeah I'd prefer not to use this method since it can change at any time. Let's just do POST and PUT and whatever else the current implementation is.